.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
   - go: tip
     env: TARGET=amd64-go-tip
   exclude:
-  - go: 1.9.3
+  - go: 1.9.4
```

Comment:
@jpbetz PTAL. Had to keep two lines for Go version. Travis somehow ignores `$TARGET_GO_VERSION` under `exclude` fields.